ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off

```

Comment:
I suspect that we shouldn't abbreviate Azure DevOps to `ado`, even in the directory tree.